tests/SDKWrapperMocker.php

```diff
@@ -182,6 +183,17 @@ public function getRequestArray(): array
 
 
 	/**
+	 *
+	 * @param onOfficeSDK $officeSDK
+	 */
+
+	public function setSDK(onOfficeSDK $officeSDK)
```

Comment:
The new setter gives the mock a different interface than the class it mocks and requires $this->_pSDK to be protected. Is there a different solution?